cypress/e2e/tests/custom-metrics/assessment_metrics.test.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    createMultipleApplications,
+    deleteApplicationTableRows,
+    login,
+} from "../../../utils/utils";
+import { Assessment } from "../../models/migration/applicationinventory/assessment";
+import { Metrics } from "../../models/migration/custom-metrics/custom-metrics";
+import { Stakeholders } from "../../models/migration/controls/stakeholders";
+import * as data from "../../../utils/data_utils";
+
+let applicationList: Array<Assessment> = [];
+let metrics = new Metrics();
+const stakeholdersList: Array<Stakeholders> = [];
+const stakeholdersNameList: Array<string> = [];
+
+describe(["@tier2"], "Custom Metrics - The total number of initiated assessments", function () {
+    before("Login and create test data", function () {
+        // Perform login
+        login();
+
+        // Navigate to Application inventory tab, delete all applications
+        deleteApplicationTableRows();
+
+        // Navigate to stakeholders control tab and create new stakeholder
+        const stakeholder = new Stakeholders(data.getEmail(), data.getFullName());
+        stakeholder.create();
```

Comment:
There is also createMultipleStakeholders() in utils , You can use that .

Reply:
Thanks!  will use `createMultipleStakeholders()`